CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/resources/quality-control-view.css

```diff
@@ -13,7 +13,7 @@
  ******************************************************************************/
 
 .table-view {
-    -fx-skin: "com.sun.javafx.scene.control.skin.TableViewSkin";
+    /*-fx-skin: "com.sun.javafx.scene.control.skin.TableViewSkin";*/
```

Comment:
I believe this still exists, it's just moved - https://openjfx.io/javadoc/14/javafx.controls/javafx/scene/control/skin/TableViewSkin.html

Reply:
updated